node/globals.ts

```diff
@@ -76,6 +76,7 @@ declare global {
     next: Maybe<string>
     report: Record<string, number>
     count: number
+    disableDraftRoutes: boolean
```

Comment:
You don't need to send this config in the event payload.
It is defined in settings, you only need to access it from ctx [next to the same place where you're reading it from event](https://github.com/vtex-apps/store-sitemap/pull/140/files#diff-59c811217644961f2482b6b8ababf962335daff8e01ea378977e28a1ef81ccbfR98). 
e.g: `const disableDraftRoutes = ctx.state.settings.disableDraftRoutes`

Reply:
Thanks @Jeymisson, we will change it to avoid sending it in the event payload.